edx_proctoring/tests/__init__.py

```diff
@@ -13,15 +12,9 @@ def setup_test_backends():
     from edx_proctoring.backends.tests.test_backend import TestBackendProvider
     from edx_proctoring.backends.null import NullBackendProvider
     from edx_proctoring.backends.mock import MockProctoringBackendProvider
-    from edx_proctoring.backends.software_secure import SoftwareSecureBackendProvider
-    extensions = [
-        Extension('test', 'openedx.proctoring', TestBackendProvider, None),
-        Extension('null', 'openedx.proctoring', NullBackendProvider, None),
-        Extension('mock', 'openedx.proctoring', MockProctoringBackendProvider, None),
-        Extension('software_secure', 'openedx.proctoring', SoftwareSecureBackendProvider, None)
-    ]
-
-    config.backends = ExtensionManager.make_test_instance(extensions)
+    config.backends['test'] = TestBackendProvider()
+    config.backends['null'] = NullBackendProvider()
+    config.backends['mock'] = MockProctoringBackendProvider()
```

Comment:
We don't need to clean these up when the test is done?

Reply:
There's probably a better way to do this. The function should run one time before the tests run. I could put it a setupClass method somewhere, but the various test suites don't inherit from a single base class.

Do you have a suggestion?

Reply:
My question was really, is it OK that these things will be in effect for all the tests?  If that's OK, then we can leave this as it is.

Pytest has a bunch of ways to manage these things if you need to clean them up.

Reply:
Yes, they apply to all tests. This monkeypatching is needed because stevedore loads the backends, and only if edx_proctoring is installed.